properties_pane/field_level/fieldLevelConfig.json

```diff
@@ -194,8 +194,6 @@ making sure that you maintain a proper JSON format.
 			"description",
 			"type",
 			"required",
-			"minProperties",
```

Comment:
Return these properties because they are supported by MongoDB spec

https://www.mongodb.com/docs/manual/reference/operator/query/jsonSchema/
https://www.mongodb.com/docs/atlas/app-services/schemas/types/#object